esp-box/esp-box.c

```diff
@@ -155,6 +155,26 @@ static void bsp_touchpad_read(lv_indev_drv_t *indev_drv, lv_indev_data_t *data)
     }
 }
 
+static void bsp_button_read(lv_indev_drv_t *indev_drv, lv_indev_data_t *data)
+{
+    uint8_t g_home_btn_val;
+    static uint8_t last_btn = 0;
```

Comment:
Does this variable ever change to 'not zero'?

Reply:
> Does this variable ever change to 'not zero'?
Probably not. It is more appropriate to delete here.